dev/io.openliberty.wsoc.21.internal_fat/fat/src/io/openliberty/wsoc/tests/BasicTest.java

```diff
@@ -0,0 +1,216 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.wsoc.tests;
+
+import static org.junit.Assert.assertNotNull;
+
+import java.io.File;
+import java.util.logging.Logger;
+
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestRule;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.ws.fat.util.browser.WebBrowser;
+import com.ibm.ws.fat.util.browser.WebBrowserFactory;
+import com.ibm.ws.fat.util.browser.WebResponse;
+
+import componenttest.annotation.AllowedFFDC;
+import componenttest.annotation.ExpectedFFDC;
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.HttpUtils;
+
+import io.openliberty.wsoc.tests.all.TimeOutTest;
+
+import io.openliberty.wsoc.util.OnlyRunNotOnZRule;
+import io.openliberty.wsoc.util.WebServerControl;
+import io.openliberty.wsoc.util.WebServerSetup;
+import io.openliberty.wsoc.util.wsoc.WsocTest;
+
+// comment #1 (referenced below): Some tests use the AllowedFFDC annotation because these Jetty client test will not run on Z, as desired
+// due to the notOnZRule, but the test framework will still look for the "ExpectedFFDC" if that annotation is used, and not finding it, will
+// fail the test on Z.
+
+// comment #2 (referenced below): Some tests we found didn't run for "test" reasons, not product code reasons, test were commented out and
+// not deleted so the same pattern would not be repeated.
+
+/**
+ * Tests WebSocket Stuff
```

Comment:
Is this a new websocket 2.1 test class or an existing class from an older websocket FAT?

Reply:
It's a test class. I renamed it to Basic21Test instead